features/unsupported/tests/mbed/basic/main.cpp

```diff
@@ -1,9 +1,30 @@
 #include "test_env.h"
+ #include "mbed.h"
 
 int main() {
-    MBED_HOSTTEST_TIMEOUT(20);
-    MBED_HOSTTEST_SELECT(default_auto);
-    MBED_HOSTTEST_DESCRIPTION(Basic);
-    MBED_HOSTTEST_START("MBED_A1");
-    MBED_HOSTTEST_RESULT(true);
+    PwmOut pwm1(D4);
```

Comment:
is this test program that you have been using for testing (because this is not basic anymore) ?